modules/nf-core/clustalo/align/main.nf

```diff
@@ -0,0 +1,49 @@
+process CLUSTALO_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::clustalo=1.2.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/clustalo:1.2.4--h87f3376_5':
+        'biocontainers/clustalo:1.2.4--h87f3376_5' }"
+
+    input:
+    tuple val(meta),  path(fasta)
+    tuple val(meta2), path(tree)
+
+    output:
+    tuple val(meta), path("*.aln"), emit: alignment
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def options_tree = tree ? "--guidetree-in=$tree" : ""
```

Comment:
is the `options_tree` used somewhere in the process or can they be supplied via args ?

Reply:
You are right, I thought it was not possible but I just tested it and it works! 
I just updated the branch.
Thanks a lot :)